src/cpp/rtps/transport/test_UDPv4Transport.cpp

```diff
@@ -192,21 +192,21 @@ bool test_UDPv4Transport::send(
 }
 
 bool test_UDPv4Transport::send(
-        const octet* send_buffer,
-        uint32_t send_buffer_size,
+        const std::array<asio::const_buffer, 3>& send_buffers,
+        uint32_t total_bytes,
```

Comment:
I do not understand why in this case you have changed the variable name when the methods called within all refer to `send_buffer_size` or `size` (in the case of `log_drop`).

Reply:
I have refactored log_drop

Reply:
Refactor in e926974cab9c6b3a8597397828a84ba076c82c2d